dom.bs

```diff
@@ -908,9 +908,14 @@ for historical reasons.
 <h3 id=interface-eventtarget>Interface {{EventTarget}}</h3>
 
 <pre class=idl>
+[Exposed=*]
+interface EventTargetInternals {
+  attribute EventTarget parent;
+};
+
```

Comment:
This feels a bit cumbersome API, one needs to pass a callback and then use its param to change a property value.

Reply:
Would you prefer an init dictionary? One downside to this, as explained in https://github.com/whatwg/dom/issues/583#issuecomment-1754494834 is that an init dictionary cannot allow setting the parent post-construction.

Reply:
Yeah, I suppose we could also do what `new Promise()` does, but it doesn't seem as extensible:
```js
new EventTarget(setParent => {
  setParent(someParent);
}
```
I think as long as the capability is only granted to whoever constructed the `EventTarget` object I'm okay, so alternative proposals welcome.